Aptoide/bds-sdk
# [APPC-1561] Adapt sdk for cafebazaar

**What does this PR do?**

   Redirects the user to install the wallet in cafe bazaar if the cafe bazaar store is installed.
   Refactored code and files package

**Where should the reviewer start?**

InstallDialogActivity.java

**How should this be manually tested?**

Install cafe bazaar store: https://cafebazaar.ir/?l=en
And without the wallet install launch the billing flow and press install, check if it redirects you to cafebazaar's wallet appview (currently isn't working)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1561

**Code Review Checklist**
